README.md

```diff
@@ -76,22 +97,23 @@ bin/python-test.sh
 ```
 However, you can also play around with CSFs directly, using the simple API examples from earlier.
 
-### Using CSFs in Python
+### Using CARAMEL in Python
 
 Our Python interface is intentionally minimal. We only expose two public methods: `caramel.CSF(keys, values)` to construct a CSF and `caramel.load(filename)` to load a CSF from a file. These methods work with a variety of supported key and value types.
```

Comment:
Let's update `caramel.CSF` to be `caramel.Caramel`

Reply:
Done